internal/pkg/deploy/cloudformation/pipeline.go

```diff
@@ -55,5 +55,5 @@ func (cf CloudFormation) UpdatePipeline(in *deploy.CreatePipelineInput) error {
 
 // DeletePipeline removes the CodePipeline stack.
 func (cf CloudFormation) DeletePipeline(stackName string) error {
-	return cf.cfnClient.Delete(stackName)
+	return cf.cfnClient.DeleteAndWait(stackName)
```

Comment:
Just noting for reviewers -- had to add this to ensure that the pipeline stack finished before the project stackset got deleted, since that was what was causing https://github.com/aws/amazon-ecs-cli-v2/issues/735.